tools/kleaver/main.cpp

```diff
@@ -95,6 +95,11 @@ namespace {
   llvm::cl::opt<std::string> directoryToWriteQueryLogs("query-log-dir",llvm::cl::desc("The folder to write query logs to. Defaults is current working directory."),
 		                                               llvm::cl::init("."));
 
+  llvm::cl::opt<bool>
+  QueryIndependence("assume-query-independence",
```

Comment:
We still have this pending discussion.  I would be fine with --clear-array-decls-after-query, but I am equally happy with --assume-query-independence.  So let's have @andreamattavelli decide.